formats/help/frontend/examples/help_page.json

```diff
@@ -0,0 +1,102 @@
+    "external_related_links": [
```

Comment:
There are no live examples of help pages with external related links however it is something that Publisher supports if it is required in the future.